### DISCUSSION TITLE
Is it possible to add attributes or remove whole nodes from the dom using ferrum?

### DISCUSSION THREAD

**Discussion by catherynl** [rubycdp/ferrum#468] [General]
First of all, loving ferrum! there's nothing else like it out there for ruby.

Question 1: I saw that `remove` was listed under the `Nodes` section -- what does that do? When I try to find a `#<Ferrum::Node>` and call `.remove` on it, I get `NoMethodError`.
EDIT: looks like that is implemented in master, probably just not in the latest release.

Question 2: I'd love to be able to add class or id attributes into the dom via ferrum -- is that possible? I guess I can evaluate custom javascript but it'd be cool to have this method.

**Comment by route** [comment]
Q1: correct it might not be released
Q2: I don't think we can implement methods for all needs, it's out of scope for Ferrum, but you can workaround it with JS
